### PR TITLE
Bug 1247774 - Port FxA L10n refactor to tv smart-system

### PR DESCRIPTION
**Pull request by zbraniecki**
https://bugzilla.mozilla.org/show_bug.cgi?id=1247774

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f6aaf4a8c3cf89a3274d24dd234d1c63dcfa48f0

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cdcc0aef3137676e2f35939a4a3b0cb28b10546d

File: tv_apps/smart-system/fxa/js/fxam_error_overlay.js [hunk 1]
```diff
@@ -60,12 +61,17 @@ var FxaModuleErrorOverlay = {
...
+      this.fxaErrorMsg.style.display = 'none';
+      this.fxaErrorMsgCoppa.style.display = 'inline';
+      this.fxaErrorMsgCoppa.setAttribute('data-l10n-id', messageL10n);
     } else {
+      this.fxaErrorMsg.style.display = 'inline';
+      this.fxaErrorMsgCoppa.style.display = 'none';
       this.fxaErrorMsg.setAttribute('data-l10n-id', messageL10n);
     }
-
+
```
Comment (begeeben) [inline tv_apps/smart-system/fxa/js/fxam_error_overlay.js]: Please remove the space before merging.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=af3ee9b2ea2fabdcd40af18e3a3858debde05c9a

**Pull request merged**